### PR TITLE
Fix date picker between

### PR DESCRIPTION
**Pull request by kukula**
Looks like this one fixes issue on front end as well

https://jobseeker.atlassian.net/browse/DEV-6255

### CODE REVIEW

File: lib/rails_admin/abstract_model.rb [hunk 1]
```diff
@@ -226,6 +226,7 @@ def last_week
         end
 
         def between
+          return [@value[1], @value[2]].sort if @value[1] && @value[2]
           [@value[1], @value[2]]
```
Comment (keang) [inline lib/rails_admin/abstract_model.rb]: how about we sort at initialization?
```
      class FilteringDuration
        def initialize(operator, value)
          @value = Array.wrap(value).sort
          @operator = operator
        end
```

Comment (kukula) [reply lib/rails_admin/abstract_model.rb]: Looks quite ugly and I think it's too much logic for `initialize`
```
 @value = Array.wrap(value).sort { |a, b| return 0 if a.nil? || b.nil?; b <=> a }   
```

Comment (keang) [reply lib/rails_admin/abstract_model.rb]: Would this work? 
```
@value = Array.wrap(value).sort
```

Comment (kukula) [reply lib/rails_admin/abstract_model.rb]: nope. we have cases with `nil`

Comment (kukula) [reply lib/rails_admin/abstract_model.rb]: https://github.com/jobseekerltd/rails_admin/blob/0758d7ffdc106603ed3a9f1224b5d8b09e92870a/spec/rails_admin/abstract_model_spec.rb#L69-L73

Comment (keang) [reply lib/rails_admin/abstract_model.rb]: ok fair enough 👍

**Pull request merged**